src/wp-includes/theme.php

```diff
@@ -1498,14 +1498,21 @@ function header_image() {
  * @return array
  */
 function get_uploaded_header_images() {
+	$stylesheet    = get_option( 'stylesheet' );
+	$transient_key = 'uploaded_header_images' . $stylesheet;
```

Comment:
A bit of a weird key because there's no underscore between the word "images" and the theme slug, let's fix that:
```suggestion
	$transient_key = "uploaded_header_images_{$stylesheet}";
```